### DISCUSSION TITLE
New series' show Options: hide_legend

### DISCUSSION THREAD

**Discussion by JSa1987** [RomRider/apexcharts-card#204] [Ideas]
Hello,

Would it be possible to add an option in the _series' show Options_ to hide the series in the legend?

This would be useful for example when background shaded areas are added to the chart to indicate status of a system. Adding these series to the chart makes it quite cluttered.

**Comment by RomRider** [comment]
Unfortunately there's no way to achieve that with ApexCharts.js :(
